lib/services/nativescript-project-plugins-service.ts

```diff
@@ -110,18 +110,23 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
 
 	public addPlugin(pluginIdentifier: string): IFuture<void> {
 		return (() => {
+			let pluginBasicInfo: IBasicPluginInformation;
 			if(this.isPluginInstalled(pluginIdentifier)) {
-				this.$logger.printMarkdown(util.format("Plugin `%s` is already installed.", pluginIdentifier));
+				this.$logger.printMarkdown(util.format("Plugin `%s` is already installed.", pluginIdentifier.toLowerCase()));
 				return;
 			}
 
-			if(this.checkIsValidLocalPlugin(pluginIdentifier).wait()) {
-				this.installLocalPlugin(pluginIdentifier, {addPluginToPackageJson: true}).wait();
+			if (path.extname(pluginIdentifier) === ".tgz") {
+				this.fetch(pluginIdentifier).wait();
```

Comment:
I've decided against adding support for `.tgz` files in this PR - this will be handled in a separate one